acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-order-line-exchange-rate-update.feature

```diff
@@ -0,0 +1,215 @@
+# For MODORDERS-1185
+Feature: Check encumbrances after order line exchange rate update
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * call loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def createInvoice = read('classpath:thunderjet/mod-invoice/reusable/create-invoice.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+  @Positive
+  Scenario: Check encumbrances after order line exchange rate update to manual rate 2 times
+    * def fundId = call uuid
+    * def budgetId = call uuid
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def invoiceId = call uuid
+    * def invoiceLineId = call uuid
+
+    ### 1. Create finances
+    * configure headers = headersAdmin
+    * def v = call createFund { id: "#(fundId)" }
+    * def v = call createBudget { id: "#(budgetId)", allocated: 10000, fundId: "#(fundId)", status: "Active" }
+    * configure headers = headersUser
+
+    ### 2. Create an order and line
+    * def v = call createOrder { id: "#(orderId)" }
+    * def v = call createOrderLine { id: '#(poLineId)', orderId: '#(orderId)', fundId: '#(fundId)' }
+
+    ### 3. Open the order
+    * def v = call openOrder { orderId: '#(orderId)' }
+
+   ###########################################################################################################
```

Comment:
as a option: we can change these lines to 
## First Manual Exchange Rate... ## Second dynamic exchange rate
to be more clear about the next scenarios

Reply:
> as a option: we can change these lines to
> 
> ## First Manual Exchange Rate... ## Second dynamic exchange rate
> to be more clear about the next scenarios

I think its not needed, both scenarios are simple enough